test/e2e/10_helm_chart.bats

```diff
@@ -18,7 +18,8 @@ function setup() {
   poll_until_equals "gitconfig secret" "$GITCONFIG" "kubectl get secrets -n $E2E_NAMESPACE gitconfig -ojsonpath={..data.gitconfig} | base64 --decode"
 
   # Apply the HelmRelease fixtures
-  kubectl apply -f "$FIXTURES_DIR/releases" >&3
+  kubectl apply -f "$FIXTURES_DIR/releases/git.yaml" >&3
```

Comment:
Moved to apply the files explictly, it was doing the entire directory before which now that I have added another release it might be safer to apply what we actually want.